### PR TITLE
added code to send message to sqs

### PR DESCRIPTION
**Pull request by vsn-hub**
-> sends message to sqs with path of s3 file in json format
-> Still needs the main resources names such as bucket and sqs url. Will add them in

### CODE REVIEW

File: internal/endpoints/fuzz.go [hunk 1]
```diff
@@ -37,17 +38,52 @@ func Fuzz(service *web.Service) {
...
+		bucketName := "test-store-swagger"
 		_, ierr := uploader.Upload(&s3manager.UploadInput{
 			//will use the bucket name as variable
-			Bucket: aws.String("test-store-swagger"),
+			Bucket: aws.String(bucketName),
 			Key:    aws.String(key),
 			Body:   bod,
 		})
 
+		s3URI := fmt.Sprintf("{\"s3_location\":\"s3://%s/%s\"}", bucketName, key)
```
Comment (The-TT-Hacker) [inline internal/endpoints/fuzz.go:49]: just pass the uuid here

File: internal/endpoints/fuzz.go [hunk 2]
```diff
@@ -37,17 +38,52 @@ func Fuzz(service *web.Service) {
...
 
+		//add sqs message
+		sess2 := session.Must(session.NewSessionWithOptions(session.Options{
+			SharedConfigState: session.SharedConfigEnable,
+		}))
+
+		svc := sqs.New(sess2)
+
+		// URL to our queue- need terraform variable here
+		qURL := "https://sqs.ap-southeast-2.amazonaws.com/117712065617/varun-test-sqs-queue"
```
Comment (The-TT-Hacker) [inline internal/endpoints/fuzz.go:63]: get this from an environment variable set by terraform

File: internal/endpoints/fuzz.go [hunk 3]
```diff
@@ -37,17 +38,52 @@ func Fuzz(service *web.Service) {
...
+
+		result, err := svc.SendMessage(&sqs.SendMessageInput{
+			DelaySeconds: aws.Int64(10),
+			MessageAttributes: map[string]*sqs.MessageAttributeValue{
```
Comment (The-TT-Hacker) [inline internal/endpoints/fuzz.go:67]: dont need these attributes

File: internal/endpoints/fuzz.go [hunk 4]
```diff
@@ -37,17 +38,52 @@ func Fuzz(service *web.Service) {
...
+			},
+			MessageBody: aws.String(s3URI),
+			QueueUrl:    &qURL,
+		})
+
+		if err != nil {
+			logger.Infof("There was an issue adding event to sqs: %s", err.Error())
+		}
+
+		logger.Infof("Success", *result.MessageId)
```
Comment (The-TT-Hacker) [inline internal/endpoints/fuzz.go:85]: Give this log message more meaning

**Comment by thefishua** [comment]
Still waiting on issues to be resolved

**Pull request closed without merging**